clarity_ext/dilution.py

```diff
@@ -220,9 +230,9 @@ def calculate_transfer_volumes(self):
                     transfer.buffer_volume *= scale_factor
                     transfer.scaled_up = True
             except (TypeError, ZeroDivisionError) as e:
-                transfer.sample_volume = None
-                transfer.buffer_volume = None
-                transfer.has_to_evaporate = None
+                transfer.sample_volume = 0
```

Comment:
I think either 0 or None will do for now. The reason I changed it was for debugging, the script crashed with assignment of None (to blank controls), and I wanted to see the ordinary output.